Resources/Audio/Items/attributions.yml

```
@@ -17,3 +17,14 @@
   license: "CC-BY-SA-3.0"
   copyright: "Time immemorial"
   source: "https://github.com/tgstation/tgstation/blob/172b533d0257fcc1f8a05406f1c9fad514c14d88/sound/items/rped.ogg"
+
+- files: ["scary_horn.ogg"]
+  license: "CC-BY-SA-3.0"
+  copyright: "theOperand"
```

```suggestion
  copyright: "Made by theOperand (github) for tgstation."
```
also if you modified them mention it (just a reminder since i havent compared the sounds)

---

made this change and also added "modified by brainfood1183 (github) for ss14" i shortened them.